website/addons/s3/static/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bucketLocations": {
+        "": "US Standard",
+        "EU": "Europe Standard",
+        "us-west-1": "California",
+        "us-west-2": "Oregon",
+        "ap-northeast-1": "Tokyo",
+        "ap-southeast-1": "Singapore",
+        "ap-southeast-2": "Sydney",
+        "cn-north-1": "Beijing"
```

Comment:
I'm not sure if boto3 supports it (but I would be surprised if they
didn't). We should begin discussing slowly porting code over. AWS (and the
boto docs) recommend starting to port as they will be deprecating boto/boto
sometime in the near future. After contacting AWS support for a more
definitive answer they said that unofficially it will be soon and that
there are no FT devs dedicated to maintaining boto/boto anymore. All they
are doing is patching critical issues and and any security related bugs.

We should be able to use both in tandem.

On Fri, Jul 31, 2015 at 11:11 AM, Michael Haselton <notifications@github.com
> wrote:

> In website/addons/s3/static/settings.json
> <https://github.com/CenterForOpenScience/osf.io/pull/3812#discussion_r35982644>
> :
>
> > @@ -0,0 +1,13 @@
> > +{
> > +    "bucketLocations": {
> > +        "": "US Standard",
> > +        "EU": "Europe Standard",
> > +        "us-west-1": "California",
> > +        "us-west-2": "Oregon",
> > +        "ap-northeast-1": "Tokyo",
> > +        "ap-southeast-1": "Singapore",
> > +        "ap-southeast-2": "Sydney",
> > +        "cn-north-1": "Beijing"
>
> Does boto3 have support, any reason not to use the latest version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/3812/files#r35982644>
> .
>



-- 
*Harry Rybacki* - Developer, Center for Open Science
Research: http://scinet.osf.io
Blog: http://rybacki.io
[image: http://github.com/harryrybacki]
<http://github.com/HarryRybacki> [image:
https://twitter.com/HarryRybacki] <https://twitter.com/HarryRybacki> [image:
http://www.linkedin.com/in/HarryRybacki/]
<http://www.linkedin.com/in/HarryRybacki/> [image:
https://plus.google.com/+HarryRybacki]
<https://plus.google.com/+HarryRybacki>